models/api.model.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs/promises')
+exports.showAllEndpoints=()=>{
+    return fs.readFile('endpoints.json', 'utf8')
+    .then((endpointData)=>{
+        return JSON.parse(endpointData)
+    })
+}
```

Comment:
A few points about this one:
- To streamline it and cut down on the code a bit, you can shortcut around having to use fs by requiring in the json file directly - this is handled well by javascript because it instantly gives us access to the file without having to use fs at all!
- Since we're not actually interacting with the database at all, this should really be in the controller instead of the model. It is the bit of the function that grabs the data, but since it's not actually making a db query, it doesn't need a model